.github/workflows/spelling.yml

```diff
@@ -92,7 +92,7 @@ jobs:
     steps:
     - name: check-spelling
       id: spelling
-      uses: check-spelling/check-spelling@v0.0.21
+      uses: check-spelling/check-spelling@v0.0.24
```

Comment:
`check-spelling` is upgraded here, but it appears the Actions runner did not catch up with the change:
```txt
Download action repository 'check-spelling/check-spelling@v0.0.21' (SHA:d7cd2973c513e84354f9d6cf50a6417a628a78ce)
```